### PR TITLE
add getting started editor-section

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/node-red/getting-started/editor/workspace.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
+---
+eleventyNavigation:
+  key: Workspace
+  parent: Editor
+meta:
+  title: Node-RED Editor Workspace
+  description: Explore the features available in the Node-RED Editor workspace component]
```
Comment (Hasmin-AC) [inline src/node-red/getting-started/editor/workspace.md:7]:
```suggestion
  description: Explore the features available in the Node-RED Editor workspace component
```

File: src/node-red/getting-started/editor/workspace.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+
+![Image showing view tools in the editor](./images/workspace-view-tool.png "Image showing view tools in the editor"){data-zoomable}
+
+The workspace provides view tools at the footer in the right corner. This includes zoom in (`Ctrl` + `+`) and zoom out (`Ctrl` + `-`) buttons to control the view of the workspace and reset the zoom level to its default.
+
+![Image showing navigator tool](./images/workspace-navigator-tool.gif "Image showing navigator tool"){data-zoomable}
+
+Additionally, it provides a view navigator that allows you to see a scaled-down view of the entire workspace. In this view, you can also see the currently visible area of the workspace in the editor. To jump to a specific workspace area, click on that area in the view navigator.
+
+If linter tool is enabled, Next to the view navigator on the left side, you can find the Linter tool option that displays the number of linting issues present in the workspace. By clicking on it, you will navigate to the Linter tool in the sidebar.
```
Comment (Hasmin-AC) [inline src/node-red/getting-started/editor/workspace.md:27]:
```suggestion
If the Linter tool is enabled, next to the view navigator on the left side, you will find the Linter tool option that displays the number of linting issues present in the workspace. By clicking on it, you will navigate to the Linter tool in the sidebar.
```

File: src/node-red/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  key: Getting Started
+meta:
+  title: Getting Started with Node-RED
+  description:  Learn the basics of Node-RED, a powerful tool for IoT integration and workflow automation.
+  keywords: node-red, node-red basics, node-red foundation, node-red editor
+---
+
+# {{ meta.title }}
+
+This section provides an overview of Node-RED, a robust visual tool designed for integrating IoT devices and automating workflows without needing extensive programming knowledge.
```
Comment (Hasmin-AC) [inline src/node-red/getting-started/index.md:12]: Should we mention the FlowFuse Platform here?

File: src/node-red/getting-started/editor/header.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+
+This option displays the changelog for the current Node-RED version in the sidebar's help tab, detailing what has been changed or fixed compared to previous versions.
+
+### About FlowFuse
+
+This option directs you to the official FlowFuse website.
+
+### FlowFuse Application
+
+This option navigates you to the FlowFuse platform.
```
[the remark this answers is not in the document]
Comment (sumitshinde-84) [reply src/node-red/getting-started/editor/header.md:277]: yes

File: src/node-red/getting-started/index.md (hunk 3 above)
Comment (sumitshinde-84) [reply src/node-red/getting-started/index.md:12]: It's Node-RED Editor guide; I don't think we should mention.

File: src/node-red/getting-started/editor/header.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Hasmin-AC) [reply src/node-red/getting-started/editor/header.md:277]: then lets mention that and link to FF cloud as well.

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**